tests/unit/src/test/scala/tests/RangeFormattingWhenSelectingSuite.scala

```diff
@@ -0,0 +1,164 @@
+package tests
+
+import munit.Location
+
+class RangeFormattingWhenSelectingSuite
+    extends BaseLspSuite("rangeFormatting") {
+  check(
+    "selecting starting the line that misindent",
```

Comment:
We are using short names like `select-starting` so that it's easier later to run single test.

Could you change them to more concise names?

Reply:
yes